lib/tasks/scrape.rb

```diff
@@ -0,0 +1,9 @@
+#Pass in a single string with your SIS Username and Password, string delimited.
+require_relative './SISScraper.rb'
```

Comment:
We should also name this file 'scrape.rake', and put all the below code in a task block.
So something like this:
```
task :scrape, [:download_dir, :username, :password] do |task, args|
  SISScraper.new(args.download_dir).download_all_sis_data(args.username, args.password)
end
```
And then you could call it with something like:
`rake scrape["../../../Downloads","slh74","IHAVETHEBESTPASSWORDS"]`